spec.bs

```diff
@@ -3900,9 +3900,15 @@ To <dfn>handle ad auction signals header value</dfn> given a [=byte sequence=] |
   1. Let |parsedSignals| be the result of [=parsing JSON bytes to an Infra value=], given
     |adAuctionSignals|.
   1. If |parsedSignals| is failure or not a [=list=], return.
-  1. [=list/For each=] |signal| of |parsedSignals|:
+  1. Let |headerAdSlots| be a new [=ordered set=].
+  1. [=list/For each=] |signal| in |parsedSignals|:
     1. If |signal| is not an [=ordered map=], [=iteration/continue=].
     1. If |signal|["`adSlot`"] doesn't exist, [=iteration/continue=].
+    1. If |headerAdSlots| [=set/contains=] |signal|["`adSlot`"], [=iteration/continue=].
+
+      NOTE: Implementations can consider also issuing a diagnostic error message in this case

```

Comment:
This note should be something not in a note, like:

> Optionally, [report a warning to the console](https://console.spec.whatwg.org/#report-a-warning-to-the-console) with a diagnostic error message indicating that duplicate headers were ignored

Or something to that effect

Reply:
Done.